<!DOCTYPE html>
<html lang="en">
<head>
<link rel="stylesheet" href="bioinfo.css" />
<title>Course resources - Download</title>
</head>
<body>
    
        <div id="menu">
            <ul class="menuitems">
        
<li class="menuitem">
 <a href="introduction_schedule.html">Introduction</a>
</li>
<li class="menuitem">
 <a href="topic_1_basic_handling_of_sequence_data.html">Topic 1 Basic handling of sequence data</a>
</li>
<li class="menuitem">
 <a href="topic_2_public_data_resources.html">Topic 2 Public Data resources</a>
</li>
<li class="menuitem">
 <a href="topic_3_sequence_similarity_searches_i.html">Topic 3 Sequence similarity searches I</a>
</li>
<li class="menuitem">
 <a href="course_resources_installations.html">Course resources</a>
<ul class="menuitems">

<li class="menuitem ">
 <a href="course_resources_installations.html">Installations</a>

<li class="menuitem ">
 <a href="course_resources_sample_data.html">Sample data</a>

<li class="menuitem ">
 <a href="course_resources_literature.html">Literature</a>

<li class="menuitem active">
 <a href="course_resources_download.html">Download</a>
</ul>
</li>
</ul>
            <h3><font color="#990000">Online tools</font></h3>
<ul class="menuitems">
  <li class="menuitem"><a href="http://kfrserver.natur.cuni.cz/lide/fatima/bioinfo/sms2/index.html" target="_blank">SMS</a></li>
  <li class="menuitem"><a href="http://bioinformatics.org" target="_blank">Bioinformatics.org</a></li>
  <li class="menuitem"><a href="http://www.ncbi.nlm.nih.gov/" target="_blank">NCBI home page</a></li>
  <li class="menuitem"><a href="http://www.expasy.org/" target="_blank">ExPaSy</a> proteomics server</li>
</ul>
</div>
    <div id="content">
<h1>
  <hr WIDTH="100%">
  <font color="#993366">Making your own copy of the course workstation</font></h1>
<h2><font color="#993366">... and some other software you might find useful</font></h2>
<hr>
<h2><font color="#990000">Instructions (in Czech) and files for setting up the
  real course workstation are <a href="course_resources_installations.html">here</a>.</font></h2>
<hr>
<h3> Color key:</h3>

<ul>
  <li>
    <p><font color="#006600">installed locally on the course workstations</font></p>
  </li>
  <li>
    <p><font color="#000099">installed on a local (Dept. Plant Physiol.) server,
      available for local installation</font></p>
  </li>
  <li>
    <p><font color="#CC0000">not installed locally/not used in the course but
      maybe interesting nevertheless</font></p>
  </li>
</ul>
<p>All programs downloadable from here are either freeware, free for educational/home
  use or free demo/trial versions. </p>

<hr width="100%">
<h2><font color="#009900">Clustal W/ Clustal X</font></h2>
<p>Clustal W is a standard multiple alignment program, written originally for
  Unix and even for VMS (anyone remembers that?), so <font color="#FF0000">lots
  of non-Windows versions are available</font>.Clustal X, though, is a Win-specific
  graphical interface. Both can be obtained <a href="http://bips.u-strasbg.fr/fr/Documentation/ClustalX/">here</a>.</p>
<p>Reference: Thompson, J.D., Higgins, D.G. and Gibson, T.J. (1994) CLUSTAL W:
  improving the sensitivity of progressive multiple sequence alignment through
  sequence weighting, position specific gap penalties and weight matrix choice.
  Nucleic Acids Research, 22(22):4673-4680.
<hr>
<h2><font color="#009900">MACAW (Multiple Alignment Construction and Analysis
  Workbench) by S. Altschul et al.</font></h2>
<p>- old but good, runs on anything from 386 with Win 3.1 upwards! No longer supported
  by NCBI where it was developed, but still around in the Net. Download <a href="ftp://ftp.ncbi.nlm.nih.gov/pub/schuler/macaw/">here</a>
  from the author´s ftp site (209 kB) and unzip to a folder on your disk (it just
  resides in its own folder without affecting the Windows registry). <font color="#FF0000">A
  Mac version exists as well.</font>
<hr>
<h2><font color="#009900">BioEdit</font></h2>
<p>- an ever better free package for alignment editing, includes a CLUSTAL interface
  and a possibility to work with translated DNA, by Tom Hall, North Carolina State
  University. <br>
  Download from the author <a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html">here</a>.
  (The author´s site experienced some problems, and may still carry only old version.
  The recent version is mirrorred on my <a href="setup/setup.html">workstation
  setup page</a>).<br>
<hr width="100%">
<h2>
<font color="#009900">pDRAW32 by Kjeld Olesen</font></h2>

<p>- a LOVELY plasmid-drawing and restriction analysis program. Technically not
  freeware but beerware, which means: if you meet the author, buy him a beer.
  If you feel that you are owing something to me, and if you happen to meet Kjeld
  Olesen, please invite him instead of me (just to increase the probablility :-)
<p>The program gets updated fairly frequently, so it is best to download directly
  from the author: <br>
  <a href="http://www.acaclone.com">http://www.acaclone.com</a> <br>
<p>&nbsp;
<p>Note: we are all growing old, including the author, apparently! Thus, the program
  has been recently upgraded from beerware to shareware, but the unpaid version
  still works. <br>
<hr width="100%">
<h2><font color="#339900">PlasmaDNA</font></h2>
<p>- a bigger, better, smarter, completely free alternative to pDraw32. Also more
  complicated and somewhat less user-friendly. Available <a href="http://research.med.helsinki.fi/plasmadna/">here</a>.
</p>
<hr width="100%">
<h2>
<font color="#009900">PHYLIP - the classical phylogeny package by Joe Felsenstein</font></h2>
Visit <a href="http://evolution.genetics.washington.edu/phylip.html">the author's
pages</a> to get the newest version for your system. <font color="#FF0000">Ported
from Unix, therefore likely to run better/faster on a Linux.</font><br>
<hr WIDTH="100%">
<h2> <font color="#CC0000">PHYML</font><font color="#009900"> </font></h2>
A program to compute maximum likelihood trees by a fast heuristic method is available
<a href="http://www.atgc-montpellier.fr/phyml/binaries.php">here</a> for download.
Compatible with PHYLIP.<br>
<hr width="100%">
<h2><font color="#009900">TreeCon - phylogenetic tree construction by Yves Van
  de Peer</font></h2>

<p>A faster and in some aspects simpler/smarter alternative to PHYLIP for construction
  of sequence-based trees. Available from the <a href="http://www.psb.rug.ac.be/bioinformatics/psb/current_projects_soft.htm">author's
  site</a> (including manuals and a new patch).
<p>Reference: Van de Peer Y, De Wachter R. (1994) TREECON for Windows:
a software package for the construction and drawing of evolutionary trees
for the Microsoft Windows environment. Computer Applications in the Biosciences,
10: 569-570.
<br>
<hr>
<h2><font color="#CC0000">TreeView X- a nice utility for plotting trees by Rod
  Page </font></h2>
<p>A good tool for displaying trees made by PHYLIP, Clustal and some other programs.
  Produces emf output that can be further edited in many graphics programs. Available
  from the <a href="http://taxonomy.zoology.gla.ac.uk/rod/treeview.html">author's
  site</a> (including manuals).<br>

<hr>
<h2><font color="#009900">DeepView (alias spdbv - the Swiss PDB Viewer)</font></h2>
<p>Viewer for 3D structures with lots of extra functions, including modelling.
  Download the install <a href="http://www.expasy.org/spdbv/">here</a> from the
  developers site, or from my <a href="setup/setup.html">local mirror (Windows
  version only)</a>. <font color="#FF0000">Available for multiple platforms.</font>
<hr>
<h2><font color="#000099">The Sequence manipulation suite by Paul Stothard</font></h2>
<p>This package will be used extensively throughout the course. Since it is javascript
  based, it is <font color="#FF0000">platform-independent</font> and does not
  have to be on the workstations - the course work will be based on a server copy
  (you can link to it and try out <a href="https://kfrserver.natur.cuni.cz/lide/fatima/bioinfo/sms2/index.html">here</a>). If you
  want to have your own copy at home or at your department server, follow the
  instructions available from the left-hand menu of SMS.
<hr>
<h2><font color="#009900">Persistence of Vision Raytracer (PoVRay)</font></h2>
<p>All you need for 3D visualisation of virtual scenes is available <a href="http://www.povray.org/">here</a>.
  Warning: it can be addictive!
<p><br>
    </div>
</body>
</html>