<!DOCTYPE html>
<head>
<link rel="stylesheet" href="bioinfo.css" />
<title>Topic 2 Public Data resources</title>
</head>
<body>
    
        <div id="menu">
            <ul class="menuitems">
        
<li class="menuitem">
 <a href="introduction_schedule.html">Introduction</a>
</li>
<li class="menuitem">
 <a href="topic_1_basic_handling_of_sequence_data.html">Topic 1 Basic handling of sequence data</a>
</li>
<li class="menuitem active">
 <a href="topic_2_public_data_resources.html">Topic 2 Public Data resources</a>
</li>
<li class="menuitem">
 <a href="topic_3_sequence_similarity_searches_i.html">Topic 3 Sequence similarity searches I</a>
</li>
<li class="menuitem">
 <a href="course_resources_installations.html">Course resources</a>
</li>
</ul>
            <h3><font color="#990000">Online tools</font></h3>
<ul class="menuitems">
  <li class="menuitem"><a href="http://kfrserver.natur.cuni.cz/lide/fatima/bioinfo/sms2/index.html" target="_blank">SMS</a></li>
  <li class="menuitem"><a href="http://bioinformatics.org" target="_blank">Bioinformatics.org</a></li>
  <li class="menuitem"><a href="http://www.ncbi.nlm.nih.gov/" target="_blank">NCBI home page</a></li>
  <li class="menuitem"><a href="http://www.expasy.org/" target="_blank">ExPaSy</a> proteomics server</li>
</ul>
</div>
    <div id="content">
      <h1> <font color="#993366">Topic 2:</font><font color="#990000"> Public data 
        resources</font> </h1>
        <h3> Sequence databases, searching, downloading. Genome sites and other "added 
            value" resources. Tools for data access and manipulation. </h3>
          <h4> 
            <hr width="100%">
          </h4>
          <h2> <font color="#000000">Where can I obtain sequence data?</font></h2>
          <ul>
            <li> <font color="#000000">Public comprehensive databases: <a href="http://www.ebi.ac.uk/embl/" target="_blank">ENA (EMBL)</a>, 
              <a href="http://www.ncbi.nlm.nih.gov/Genbank/index.html" target="_blank">GenBank</a>, 
              <a href="http://www.ddbj.nig.ac.jp/" target="_blank">DDBJ</a> (DNA + protein)</font></li>
            <li> <font color="#000000">Public specialized/selective databases - examples:</font></li>
            <ul>
              <li> <font color="#000000">by molecule type + additional available data (<a href="http://www.uniprot.org/" target="_blank">UniProt</a>)</font></li>
              <li> <font color="#000000">by organism e.g. <a href="https://plantgenie.org/" target="_blank">Plant 
                GenIE</a>, <a href="https://bar.utoronto.ca/thalemine/begin.do" target="_blank">ThaleMine</a></font></li>
              <li><font color="#000000">by funding agency/institution that has sequenced 
                the genomes (<a href="http://www.jgi.doe.gov/" target="_blank">DOE Joint 
                Genome Institute</a>)</font></li>
            </ul>
            <li> <font color="#000000">... of course, experimentally, and then you should 
              deposit to the public databases.</font></li>
          </ul>
          Selected resources will be demonstrated and used in the practical tasks. <br>
          <hr width="100%">
          <h3> The <a href="http://www.ncbi.nlm.nih.gov" target="_blank">NCBI</a> 
            portal</h3>
          - more than just sequences: 
          <ul>
            <li> Literature - PubMed/MEDLINE + some online journals + books</li>
            <li> GenBank</li>
            <li> 3D structures</li>
            <li> Genomes</li>
            <li> Links to similarity searches (seeLesson 3<a href="search1.html"></a>)</li>
          </ul>
          <h4> Searching GenBank:</h4>
          <ul>
            
            <li>database subsets - DNA (incl. EST, STS, GSS ...), protein - Limits function 
            </li>
            <li>unique accession numbers </li>
            <li>anatomy of a GenBank record, saving options ... <br>
            </li>
          </ul>
          <hr>
          <h3> <a href="http://www.ebi.ac.uk/services/" target="_blank">EMBL/EBI</a> database portal</h3>
          <ul>
            <li> a smart but complicated way to access EMBL (has a good help)</li>
          </ul>
          <hr width="100%">
          <h3> Genome sites - the Arabidopsis example:</h3>
          <ul>
            <li> <a href="http://www.arabidopsis.org/" target="_blank">TAIR </a>... (most 
              contents now paid but have a look e.g. at the Insertion, Knockout and Mutation 
              resources!)</li>
            <li><a href="http://signal.salk.edu/" target="_blank">SIGnAL</a> </li>
            <li><a href="http://www.plantgdb.org/AtGDB/" target="_blank">AtGDB</a></li>
            <li><font color="#000000"><a href="https://bar.utoronto.ca/thalemine/begin.do" target="_blank">ThaleMine</a></font></li>
          </ul>
          <p>A frequently used general format for displaying and organizing genome information:<a href="https://jbrowse.org/" target="_blank"> 
            JBrowse </a></p>
          <hr width="100%">
          <h3> Transcriptome sites (demo):</h3>
          <ul>
            <li>EBI <a href="http://www.ebi.ac.uk/arrayexpress/" target="_blank">ArrayExpress</a></li>
            <li>Gene Expression Omnibus (<a href="http://www.ncbi.nlm.nih.gov/geo/" target="_blank">GEO</a>)</li>
            <li>ePlant and eFPBrowser ("electronic fluorescent protein browser"), for multiple 
              plant species is part of then <a href="http://bar.utoronto.ca/" target="_blank">BAR 
              Tools</a>)</li>
            <li><a href="https://genevestigator.com/" target="_blank">Genevestigator</a> 
              (only very basic functions are free, multiple organisms)<br>
            </li>
          </ul>
          <hr style="width: 100%; height: 2px;">
          <h3>Ontology:</h3>
          <p>Controlled dictionary to designate biological processes, structures, conditions, 
            etc. </p>
          <ul>
            <li>Biomart: tool to bulk download genetic data, with the option to apply filters. 
              (<a href="https://www.ensembl.org/biomart/martview" target="_blank">general
              </a>and <a href="http://plants.ensembl.org/biomart/martview" target="_blank">plant</a> 
              version, help and tutorials)</li>
            <li>The <a href="http://www.pantherdb.org/" target="_blank">PANTHER</a> classification 
              system<br>
            </li>
          </ul>
          <hr style="width: 100%; height: 2px;">
          <h3>Last but not least: finding literature:</h3>
          <ul>
            <li>NCBI - PubMed<br>
            </li>
            <li> <a href="http://scholar.google.cz/" target="_blank">Google Scholar</a></li>
            
            <li><a href="http://apps.webofknowledge.com" target="_blank">Web of Science</a> (paid - University has access)</li>
          </ul>
          <hr width="100%">
          <p></p>
          <h2> <a name="tasks"></a>Tasks</h2>
          <h3> 2.1</h3>
          Obtain and inspect the sequence of the pGWB4 cloning vector. Examine the various 
          sequence formats available and the annotations. Using the knowledge from the previous 
          lesson, construct <b><font color="#cc33cc">a map of pGWB4</font></b>. 
          <hr width="100%">
          <h3> 2.2</h3>
          Search the protein section of the GenBank/EMBL database for poplar<i> (Populus 
          </i>sp<i>.)</i> extensins. 
          <ul>
            <li> Save at least five of the sequences you have found as a "multiple" FASTA 
              file and keep them for future use. What is peculiar about the extensins?</li>
            <li> Select one of the sequences and explore the "Related information" section (to the right 
              from sequence description). Find the corresponding nucleotide sequence and 
              examine the sequence file.</li>
          </ul>
          <hr width="100%">
          <h3> 2.3</h3>
          Search the protein section of the GenBank/EMBL database for Rab GDP dissociation 
          inhibitors of <i>Arabidopsis thaliana</i>. 
          <ul>
            <li> Download all the "Refseq" sequences you have found, save them as a "multiple" 
              FASTA file, discard sequences that are not GDI )and keep the rest for future 
              use. How many different RabGDIs did you find?</li>
            <li> Explore the annotation of these sequences, paying attention to the locus 
              identifiers (either in the "At1g59810" - i.e. "genomic" -&nbsp; or the "F21G12.120" 
              - "BAC" - format). Use the genomic identifiers to annotate the sequences in 
              the FASTA file produced in the previous step.Keep the file for future use.</li>
            <li><b>Demo: </b>Visit the <a href="http://signal.salk.edu/" target="_blank">SIGnAL</a> 
              database and search for available information on the loci you have found, 
              using the genomic identiiers for a&nbsp; "Gene search". .Examine the expression 
              of at least one of the found loci using <a href="http://bar.utoronto.ca/" target="_blank">BAR</a> 
              tools.</li>
          </ul>
          <hr width="100%">
          <h3>2.4 </h3>
          <p>Use the <a href="http://plants.ensembl.org/biomart/martview/" target="_blank">plant 
            BIOMART tool</a> to, find all genes that are actin binding (using gene ontology 
            term) in the Arabidopsis thaliana (TAIR10) database.. In the final gene list, 
            find the number of genes and<b><font color="#0000FF"> <font color="#006600">identify 
            at least 3 gene families, whose members bind actin in <i>Arabidopsis thaliana</i></font></font></b>.<br>
            <br>
          </p>
          <p>&nbsp;</p>        
    </div>
</body>
</html>