<!DOCTYPE html>
<head>
    <title>Topic 1 Basic handling of sequence data</title>
    <link rel="stylesheet" href="bioinfo.css" />
</head>
<body>
    
    
        <div id="menu">
            <ul class="menuitems">
        
<li class="menuitem">
 <a href="introduction_schedule.html">Introduction</a>
</li>
<li class="menuitem active">
 <a href="topic_1_basic_handling_of_sequence_data.html">Topic 1 Basic handling of sequence data</a>
</li>
<li class="menuitem">
 <a href="topic_2_public_data_resources.html">Topic 2 Public Data resources</a>
</li>
<li class="menuitem">
 <a href="course_resources.html">Course resources</a>
</li>
</ul>
            <h3><font color="#990000">Online tools</font></h3>
<ul class="menuitems">
  <li class="menuitem"><a href="../sms2/index.html" target="_blank">SMS</a></li>
  <li class="menuitem"><a href="http://bioinformatics.org" target="_blank">Bioinformatics.org</a></li>
  <li class="menuitem"><a href="http://www.ncbi.nlm.nih.gov/" target="_blank">NCBI home page</a></li>
  <li class="menuitem"><a href="http://www.expasy.org/" target="_blank">ExPaSy</a> proteomics server</li>
</ul>
</div>







    <div id="content">
        
        <h1><font color="#990000">Topic 1: Basic handling of sequence data</font></h1>
      <h2> <font color="#000000">Introduction </font></h2>
      <ul>
        <li><font color="#000000">use of course workstations - data handling </font><b><font color="#FF6666">(use 
          either your folder on the server or your own USB stick).</font></b> </li>
      </ul>
      <h4>
        <hr WIDTH="100%">
      </h4>
      
      <h2> <font color="#009900">What do we have</font> - and&nbsp; <font color="#FF0000">what 
        we don't have</font>?</h2>
      
      <ul>
        <li> 
          <p><font color="#009900">a computer connected to the Internet (incl. Web browser)</font></p>
        </li>
        <li> 
          <p><font color="#009900">a text editor (Notepad or better)</font></p>
        </li>
        <li> 
          <p><font color="#009900">public databases of genomic sequences, cDNA + EST,</font><font color="#009900"> 
            protein sequences, structures and motifs,</font><font color="#009900"> gene 
            expression data etc.</font></p>
        </li>
        <li> 
          <p><font color="#FF0000">money for specialised software packages</font></p>
        </li>
        <li> 
          <p><font color="#009900">public servers capable of (almost) anything we wish 
            to do</font></p>
        </li>
      </ul>
<ul>
  <li><font color="#000000">Identification of relevant sequence portions, file 
    formats, reformatting utilities.</font></li>
  <li><font color="#000000"> Restriction site analysis and translation of DNA 
    sequences.</font></li>
  <li><font color="#000000">Construction of graphical maps, <i>in silico</i> cloning.</font></li>
</ul>
<h4>
  <hr width="100%">
</h4>

<h2> Sequence data:</h2>

types (protein, DNA, RNA), <a href="http://bioinformatics.org/seqext/iupac.html" target="_blank">IUPAC 
codes</a>, conventions. 
<h3>
Other data:</h3>
genomic maps, 3D structure files, alignments, patterns, microarray hybridization
data ... we'll come to them.
<hr width="100%">
<h3>
Sequence file formats:</h3>

<ul>
  <li>
    <p> Make sure you have the correct format.</p>
  </li>
  <li> 
    <p><a href="http://www.ncbi.nlm.nih.gov/blast/fasta.shtml" target="_blank">FASTA 
      format</a> is (almost) always correct.</p>
  </li>
</ul>

<blockquote><font face="Courier New, Courier, mono">&gt;sequence_name anything 
  behind the first gap is a comment<br>
  thisisasequenceinfastaformat</font></blockquote>

<ul>
  <li>
    <p> If not, you can (almost) always use raw data.</p>
  </li>
  <p><font face="Courier New, Courier, mono">thisisasequenceinrawformat</font>
</p></ul>

<ul>
  <li> 
    <p> There are more formats, such as <a href="http://www.genomatix.de/online_help/help/sequence_formats.html" target="_blank">EMBL, 
      GenBank and others</a> such as <a href="http://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=1457811&amp;dopt=Abstract" target="_blank">those 
      generated by automatic sequencers</a>, but you don't need to know how to 
      write these by hand!</p>
  </li>
  <li> 
    <p> If things do not work, check for <b>gaps in sequence</b>, <b>empty lines</b>, 
      and the <b>file extension</b> (some programs are sensitive).</p>
  </li>
  <li> 
    <h3><font color="#ff6600">Beware of Microsoft!</font></h3>
  </li>
</ul>

<hr width="100%">
<h3>
Converting sequences from one format to another:</h3>

<ul>
  <li>
    <p> by hand - boring!</p>
  </li>
  <li>
    <p> utilities - network or local (<a href="../sms2/index.html" target="_blank">SMS</a>; 
      we are using a LAN server in the course). These can do other things as well, 
      such as reverse complement, translation, restriction analysis etc. A variety 
      of other utilities can be found <a href="http://molbiol-tools.ca/" target="_blank">here</a>, 
      feel free to explore.</p>
  </li>
</ul>

<hr width="100%">
<h3>
Orientation in a sequence:</h3>

<ul>
  <li>
    <p> translation</p>
  </li>
  <li>
    <p> restriction mapping</p>
  </li>
</ul>

<hr width="100%">
<h3> In silico cloning: </h3>
<ul>
  <li>PlasmaDNA, SerialCloner (local applications have to be installed) </li>
  <li>Nice online tools here (<a href="http://www.molbiotools.com/" target="_blank">Molbiotools</a>)</li>
  <li><a href="http://www.bioinformatics.org/savvy/" target="_blank">SAVVY</a> 
  </li>
</ul>
<hr width="100%">
<h2>
<a name="tasks"></a>Tasks</h2>

<h3> In every lesson, some (partial) tasks are described in <font color="#cc33cc">c</font><font color="#3366ff">o</font><font color="#00cc00">l</font><font color="#ffcc00">o</font><font color="#ff0000">r</font>. 
  These are "control points" required for getting course credits in the Faculty 
  of Sciences course. At least four "colored" tasks, each of a different color, 
  have to be presented. </h3>
<h3><font color="#cc33cc">NEW: report of a dead link</font><font color="#3366ff"> 
  (or links) from the Links list <font color="#00cc00">(left bottom</font></font><font color="#00cc00"> 
  corner of the page)</font> <font color="#ffcc00">counts as a replacement</font><font color="#ff0000"> 
  for any of the colors. </font> 
  <hr width="100%">
</h3>

<h3>
1.1</h3>
You have just sequenced an Arabidopsis cDNA fragment cloned in the EcoRI site 
of the <a href="http://www.addgene.org/vector-database/1947/" target="_blank">pBluescriptII 
SK-</a> vector and obtained the following raw sequence: 
<p><font face="Courier New, Courier, mono">CCGCGGTGGCGGCCGCTCTAGAACTAGTGGATCCCCCGGGCTGCAGGAAT 
  <br>
  TCGGTGACCCCGGCAAAGCTTGCTTAATCCGAAGACGTTTCTGTTTCATC <br>
  TTCTTAAATCCGGGCCAACNGCGTTTACGAGACTAAACGCGTTTCTCTTT <br>
  AGGGCTTAATTATTATCCAGAGATGGCTCATCATAGCAAATGTTTACAAA <br>
  CGTTGGATTTAGCTTGTAAAGAGCTGAGATCTCGTGGCTTGTTTGTGAAG <br>
  CTTTTGGAGGCAATACTTAAAGCTGGAAACAGAATGAACGCGGGTACCGC <br>
  GAGAGGAAACGCTCAAGCGTTTAATCTAACCGCGCTTTTGAAGCTTTCGG <br>
  ATGTTAAAAGCGTTGATGGGAAGACTTCTTTGCTTAACTTTGTAGTGGAG <br>
  GAAGTTGTTAGATCGGAAGGAAAACGTTGTGTTATGAATAGAAGAAGCCA <br>
  TAGCTTAACACGAAGCGGTAGTAGTAACTACAATGGTGGTAATAGTAGTC <br>
  TTCAGGTTATGTCGAAAGAAGAGCAAGAGAAAGAGTACTTGAAGCTTGGT <br>
  TTACCAGTTGTTGGTGGATTGAGCTCTGAGTTTTCAAACGTGAAGAAAGC <br>
  TGCTTGTGTGGACTATGAAACGGTTGTTGCAACTTGTTCTGCTCTTGCGG <br>
  TTAGAGCGAAAGATGCGAAAACGGTGATTGGAGAATGTGAAGATGGAGAA <br>
  GGAGGGAGGTTTGTGAAAACGATGATGACGTTTCNTGATTCGGTAGAGGA <br>
  AGAGGTGAAAATAGCGAAAGGTGAAGAGAGGAAAGTGATCCCTGA </font><br>
  &nbsp;
</p><ol>
<li>
Copy the sequence into a text file and convert it to the FASTA format.</li>

  <li> Perform restriction analysis to identify sites from the vector polylinker 
    and remove vector sequences so that you have only clean cDNA left (Recommended 
    program: SMS Restriction Digest). Check the vector-insert boundary using the 
    map provided below and remove vector sequences. Save the clean cDNA sequence 
    (in FASTA format) as a text file.</li>
</ol>
<img src="bluescript.JPG" height="144" width="966"><br>
<hr width="100%">
<h3>
1.2</h3>
You have just sequenced the following&nbsp; Arabidopsis cDNA fragment.
Note the format of the sequence: a header line (<tt>&gt;mysequence</tt>) followed
by the rest of the sequence, which means FASTA format.
<p><font face="Courier New, Courier, mono">&gt;cDNA_2_At <br>
  CAAGCAACCTTTACACATATAGAAGAAGAAAAACACTTCTTTGTTTCTGTCATTAATTCCCTCCCTCTAT <br>
  ATATATATATTTAAATCTATTATGACAAAACAATCCAATACTGGATACTTTTTACAACAACATGCACAGA <br>
  CAAAGCTGAGATCTCACCTTAAGAAACTAATGAGATTGAGTTATGTTCGTCTTCATCAGACGAAGAGGTT <br>
  GAAGACGACGACGAAGAAGAAGAAGATTGTCTTCGTCCAACGAGTCCAGGAAGAGGTTGTGGCATCATTG <br>
  GATTCACTGGAACAGGAAACTTATGAGCAGAACTAACCATTGTTCTTTCGTTTATCATCCCTACTTCTTT <br>
  GCAAACTCTGTCTACTACTCCAAGGAAGTCTCTAACCACCAAGAATATTCTAAACGGATGCGCTTCTTCT <br>
  TTAGCCGAGTTTCCATGGAAATACTCTGTGATTTCTTTTACAAGTGATAACGCTACGCTCTCTTGAGCTT <br>
  GTACTCTGATGATCTCTTCCTCAGCTCTTTTCAGAAACNTTTTCATCGATTCCGNNAACCTCTGACTGTT <br>
  GCTTTCTTCTGTGATTGTTGATTGGACTTGGATTGCTTCGTTGATCTTGGCAATGNNTTGAGGAAAGCTT <br>
  GGAGANGTAGCTGCTTAGTACTTCTGAGTCCATCNNAGC </font>
</p><ol>
  <li> Convert the sequence to antisense (reverse complement) and keep it as a 
    FASTA file using SMS Reverse Complement.</li>

  <li> Look at the coding capacity of your cDNA in both directions (using SMS 
    ORF finder). Select the longest reading frame and keep its protein translation as a FASTA file. 
    (NOTE: you might not necessarily get a reasonable ORF - assume that this was 
    a raw first run sequence, containing possible frameshifts and other errors).</li>
</ol>

<hr width="100%">
<h3>
1.3</h3>
The following DNA sequence was assembled from two files, each of them a
different format.
<p><font face="Courier New, Courier, mono">&nbsp;&nbsp; 1 TTTAATAAAA TAAAAATCCA 
  CTCGCATTTT TATTTTCAAC ATTGTGCGTA&nbsp;&nbsp;&nbsp; 50 <br>
  &nbsp; 51 CGGTGCAATT CAATGAACAG TGTTTACTTT CAGTGTGTAC ACTTCTGCGG&nbsp;&nbsp; 
  100 <br>
  &nbsp;101 ACTATTACAA AGTCCACGTC TTATCCTACG TGTTATAATC TCATATGTTA&nbsp;&nbsp; 
  150 <br>
  &nbsp;151 CTGTCTGAAA TGGACCCCAC TACGTAAAAA TAAAATTAAG AATCAACCAC&nbsp;&nbsp; 
  200 <br>
  &nbsp;201 TCTTCTTCCA TCACCTCTTT TGGCTTTCTC TCTACTCTCT CTACTACTCT&nbsp;&nbsp; 
  250 <br>
  &nbsp;251 CTCACCATCA CTGAGTTAAG AGAACAAACC AAAAACAAAA TTATCAAACC&nbsp;&nbsp; 
  300 <br>
  &nbsp;301 ATCACCAGCA GAATCTTAGC TGGATTCATC ACTCTATTCA AAAAGTTTCT&nbsp;&nbsp; 
  350 <br>
  &nbsp;351 CTCTTCTCTT TTCTCAGATC TTGAACTCTT GAAGAAGAAA GAAGAAGATA&nbsp;&nbsp; 
  400 <br>
  &nbsp;401 ACACAATGCT CTTCTTCTTA TTCTTCTTCT ACTTACTCTT ATCTTCATCC&nbsp;&nbsp; 
  450 <br>
  &nbsp;451 TCCGATCTAG TCTTCGCCGA CCGTCGTGTA CTCCACGAAC CATTCTTCCC&nbsp;&nbsp; 
  500 <br>
  &nbsp;501 TATAGATTCA CCACCACCGT CACCACCATC ACCACCACCA CTTCCTAAAC&nbsp;&nbsp; 
  550 <br>
  &nbsp;551 TACCATTCTC TTCAACCACT CCTCCATCTT CATCAGACCC AAATGCTTCT&nbsp;&nbsp; 
  600 <br>
  &nbsp;601 CCTTTCTTCC CTTTATACCC TTCATCTCCA CCACCACCTT CTCCAGCCTC&nbsp;&nbsp; 
  650 <br>
  &nbsp;651 CTTCGCTTCT TTTCCGGCGA ATATCTCATC TCTAATCGTC CCTCACGCCA&nbsp;&nbsp; 
  700 <br>
  &nbsp;701 CTAAATCCCC ACCTAACTCC AAAAAACTCC TTATCGTCGC TATCTCCGCC&nbsp;&nbsp; 
  750 <br>
  &nbsp;751 GTTTCCTCCG CTGCTTTAGT CGCTCTACTT ATCGCTTTAC TCTATTGGCG&nbsp;&nbsp; 
  800 <br>
  &nbsp;801 AAGAAGCAAA CGTAACCAAG ATCTTAACTT CTCCGATGAT AGCAAAACAT&nbsp;&nbsp; 
  850 <br>
  &nbsp;851 ACACCACCGA CAGTAGCCGC CGTGTCTACC CTCCTCCTCC GGCAACGGCG&nbsp;&nbsp; 
  900 <br>
  &nbsp;901 CCTCCAACAC GACGCAATGC GGAGGCTAGA AGTAAACAGA GGACCACCAC&nbsp;&nbsp; 
  950 <br>
  &nbsp;951 GAGCTCCACC AATAACAACA GCTCTGAGTT TCTTTACTTA GGAACAATGG&nbsp; 1000 
  <br>
  1001 TGAATCAAAG AGGAATCGAT GAACAATCTC TTAGTAATAA TGGATCAAGC&nbsp; 1050 <br>
  1051 TCAAGAAAAC TTGAATCTCC AGATCTTCAA CCACTTCCTC CATTGATGAA&nbsp; 1100 <br>
  1101 ACGAAGTTTC CGTTTAAATC CAGATGTTGG TTCAATCGGA GAAGAAGATG&nbsp; 1150 <br>
  1151 AAGAAGATGA GTTTTACTCT CCACGTGGCT CACAAAGCGG GCGAGAACCG&nbsp; 1200 <br>
  1201 TTAAACCGGG TCGGACTTCC GGGTCAAAAT CCTAGATCTG TTAACAATGA&nbsp; 1250 <br>
  1251 CACTATCTCT TGCTCATCTT CAAGCTCTGG TTCACCAGGA AGATCAACAT&nbsp; 1300 <br>
  1301 TTATCAGTAT CTCTCCTTCA ATGAGTCCTA AGAGATCTGA ACCAAAACCG&nbsp; 1350 <br>
  1351 CCGGTTATCT CCACACCAGA ACCGGCGGAG TTAACCGATT ATAGATTTGT&nbsp; 1400 <br>
  1401 TCGGTCTCCG TCACTGTCGT TAGCTTCTTT ATCGTCGGGA TTGAAAAACT&nbsp; 1450 <br>
  1451 CCGATGAAGT AGGATTGAAT CAAATCTTTA GATCTCCGAC GGTTACATCT&nbsp; 1500 <br>
  1501 CTAACAACTT CACCGGAGAA TAACAAAAAA GAGAACTCTC CATTATCATC&nbsp; 1550 <br>
  1551 TACTTCAACT TCACCGGAAC GACGACCAAA TGATACACCA GAAGCTTACT&nbsp; 1600 <br>
  TGAGATCTCCGTCGCATTCTTCTGCTTCTACATCACCGTATAGATGTTTT <br>
  CAGAAATCTCCGGAGGTCTTACCGGCGTTTATGAGTAATCTCCGGCAAGG <br>
  TTTGCAATCTCAGTTACTATCTTCTCCTTCTAACTCTCATGGAGGACAAG <br>
  GTTTCCTTAAGCAGTTAGATGCATTACGTTCTCGTTCACCGTCGTCGTCT <br>
  TCTTCTTCTGTTTGTTCTTCACCGGAGAAAGCTTCTCATAAGTCACCAGT <br>
  TACATCTCCTAAGTTATCTTCCCGGAATTCGCAGTCTCTATCATCTTCTC <br>
  CGGATAGAGATTTTAGTCATAGCTTAGATGTATCACCACGGATATCGAAC <br>
  ATTTCACCTCAAATTTTACAGTCTCGTGTGCCTCCGCCTCCTCCTCCTCC <br>
  CCCACCGTTGCCGTTGTGGGGACGACGGAGTCAGGTGACTACTAAAGCGG <br>
  ACACAATCTCGAGACCGCCTTCTCTTACACCGCCTTCACATCCTTTTGTG <br>
  ATCCCATCTGAAAACTTACCAGTGACTTCGTCTCCTATGGAGACTCCAGA <br>
  GACGGTTTGTGCGAGTGAGGCGGCGGAGGAAACTCCGAAACCGAAGCTAA <br>
  AGGCGTTACATTGGGATAAAGTTAGAGCAAGTTCGGATCGTGAGATGGTT <br>
  TGGGATCATCTTCGATCAAGCTCTTTCAAGTGAGTTAATGTGACATACTC <br>
  GTTTATATGATACTATATGCTTTTAGTGAGAATGTGGTTGTTGAGATTAT <br>
  GAATGTGGTTTGCAGATTAGATGAGGAGATGATTGAGACGTTGTTTGTGG <br>
  CGAAGTCGTTAAACAACAAACCAAATCAGAGTCAGACAACTCCAAGATGT <br>
  GTTCTCCCGAGCCCGAACCAAGAGAACAGAGTCCTGGACCCGAAGAAGGC <br>
  TCAGAATATTGCCATCTTGCTTCGTGCACTTAATGTCACTATAGAAGAAG <br>
  TTTGTGAGGCTCTTCTTGAAGGTAAACTATGCTGTCACATACATAGTTTC <br>
  TCATTTTCTTCTCCTTTGATCTCCAGAATTAGAGTTCTTATGCATTTGTT <br>
  AATGGTTTTTCGATGATATGGTTGAGTTATTCTGAAAGCTTTGCTTCTTT <br>
  GATGGTGTGGAGATTCTTGGTTACATTGATGTTCTTAGTTATGCTTTTTC <br>
  AGGCAATGCTGATACACTGGGGACTGAACTTCTTGAGAGCTTACTGAAGA <br>
  TGGCACCGACAAAAGAAGAAGAGCGCAAGTTGAAAGCGTACAATGATGAT <br>
  TCGCCTGTTAAGCTTGGACATGCTGAGAAATTCCTTAAGGCAATGTTGGA <br>
  CATCCCTTTCGCCTTTAAAAGAGTTGATGCAATGCTCTATGTAGCCAACT <br>
  TTGAGTCCGAGGTTGAATACTTGAAGAAATCTTTTGAGACTCTTGAGGTA <br>
  TATATTACAAGCTATTCTCTCTCTTTTTACCATATGGTTGTATTGTAACA <br>
  GATTATGACTTCATTTCTATTGTTTGTGTAGGCTGCTTGTGAAGAACTGA <br>
  GGAACAGTAGGATGTTCTTAAAGCTTCTTGAAGCGGTTCTAAAGACAGGA <br>
  AACCGTATGAACGTTGGAACAAACCGAGGAGATGCACATGCGTTCAAGCT <br>
  TGATACACTTCTCAAGCTAGTCGATGTCAAAGGCGCTGATGGGAAAACAA <br>
  CTCTCTTGCATTTCGTTGTACAAGAGATAATCCGAGCAGAAGGCACACGT <br>
  CTCTCAGGTAACAATACACAAACAGATGACATTAAATGCCGGAAACTAGG <br>
  TCTCCAAGTTGTATCAAGTCTCTGTTCTGAGCTTAGTAACGTCAAGAAAG <br>
  CTGCTGCGATGGACTCAGAAGTACTAAGCAGCTACGTCTCCAAGCTTTCT <br>
  CAAGGCATTGCCAAGATCAACGAAGCAATCCAAGTCCAATCAACAATCAC <br>
  AGAAGAAAGCAACAGTCAGAGGTTTTCGGAATCGATGAAAACGTTTCTGA <br>
  AAAGAGCTGAGGAAGAGATCATCAGAGTACAAGCTCAAGAGAGCGTAGCG <br>
  TTATCACTTGTAAAAGAAATCACAGAGTATTTCCATGGAAACTCGGCTAA <br>
  AGAAGAAGCGCATCCGTTTAGAATATTCTTGGTGGTTAGAGACTTCCTTG <br>
  GAGTAGTAGACAGAGTTTGCAAAGAAGTAGGGATGATAAACGAAAGAACA <br>
  ATGGTTAGTTCTGCTCATAAGTTTCCTGTTCCAGTGAATCCAATGATGCC <br>
  ACAACCTCTTCCTGGACTCGTTGGACGAAGACAATCTTCTTCTTCTTCGT <br>
  CGTCGTCTTCAACCTCTTCGTCTGATGAAGACGAACATAACTCAATCTCA <br>
  TTAGTTTCTTAAGGTGAGATCTCAGCTTTGTCTGTGCATGTTGTTGTAAA <br>
  AAGTATCCAGTATTGGATTGTTTTGTCATAATAGATTTAAATATATATAT <br>
  ATAGAGGGAGGGAATTAATGACAGAAACAAAGAAGTGTTTTTCTTTTCTG <br>
  CATTTGTGTAAAAAAAATAATATAGGTTTACCTTAAAATTTGTTCATCTT <br>
  AAATTAATAATTTAAGAATCAAATAAATTTGTTTATCTGAACCGTGTGTA <br>
  CCACGAAAGAATGTGAGAGCAAACATATTACTTACTTACCCTTCGTTGCT <br>
  GAATATAATGATCATTATAAATCACTACCTCCAGTACCTTCTACCTTCTT <br>
  CAAAGAACCTTGTTGGATTTGAACCAAAGTTGGAACATAATTGACGAGAG <br>
  GTGAGCATCTAGATTCTGCATCGTGATGATGATCCACTTTTATCTATTTA </font>
</p><ol>
  <li> Convert the sequence to FASTA (recommended: SMS Filter DNA).</li>

  <li> Construct its restriction map using SMS Restriction Map, <a href="http://bio.lundberg.gu.se/cutter2/" target="_blank">WebCutter</a>.or 
    Molbiotools Restriction Analyzer</li>

  <li> A "virtual cloning" excercise: Take the&nbsp; EcoRI-XhoI fragment and clone 
    it into pBluescript SK- (sequence and map can be found here 
    in the <a href="../data/pBluescript.pof">PlasmaDNA format</a> . Create a <b><font color="#cc33cc">map of the product</font></b>
using PlasmaDNA. Save any fasta intermediates as *.fasta.</li>
</ol>

<hr width="100%">
<h3>1.4</h3>
<p>Using SAVVY, 
  construct <b><font color="#cc33cc">a map of a plasmid, pTEST</font></b>, of 
  the following properties:</p>
<ul>
  <li>total size 4.7 kb</li>
  <li>insert "MyGene" with 5´end defined by an EcoRI site at position 
    1 and 3´end at a SacI site at position 1950 bp</li>
  <li>bacterial origin of replication ("Ori") between position 2800 
    and 3100 .</li>
  <li>kanamycin resistance "KanR" between position 3400 and 4500 in 
    the reverse orientation.</li>
</ul>
<p>Hints for saving the output:</p>
<ul>
  <li>the best way is printing to a *.pdf file using any PDF printer (such as, 
    e.g., the free <a href="http://www.primopdf.com/" target="_blank">PrimoPDF</a>)</li>
  <li>if you have no PDF printer but can print to Microsoft XPS, the resulting 
    file can be converted to *.pdf <a href="http://xps2pdf.co.uk/" target="_blank">here</a></li>
  <li>if you cannot print to file,save the *.svg file and view it in a browser</li>
</ul>
    
</body>
</html>